packages/psss/src/views/Tabs/WeeklyCasesTabView.js

```diff
@@ -61,64 +51,36 @@ const DateSubtitle = styled(Typography)`
   color: ${props => props.theme.palette.text.secondary};
 `;
 
+const getCountryWeekData = (data, activeWeek) => data.find(c => c.period === activeWeek);
+
+const NUMBER_OF_WEEKS = 10;
+
 export const WeeklyCasesTabViewComponent = React.memo(({ handleOpen, activeWeek }) => {
-  const [page, setPage] = useState(0);
   const { countryCode } = useParams();
-  const {
-    isLoading,
-    error,
-    data,
-    order,
-    orderBy,
-    handleChangeOrderBy,
-    isFetching,
-  } = useTableQuery('country-weeks', { countryCode });
+  const defaultPeriod = getCurrentPeriod('WEEK');
 
-  // Todo: Get the latest week data when there is real data
-  const latestWeek = data ? data.data[0] : [];
+  const { isLoading, error, data } = useCountryWeeklyReport(
+    countryCode.toUpperCase(),
+    defaultPeriod,
+    NUMBER_OF_WEEKS,
+  );
 
   return (
     <Container>
       <Main data-testid="country-table">
         <CountryTable
-          data={data ? data.data : []}
+          data={data}
           isLoading={isLoading}
           errorMessage={error && error.message}
-          order={order}
-          orderBy={orderBy}
-          onChangeOrderBy={handleChangeOrderBy}
-          page={page}
```

Comment:
I've created another card for hooking up the pagination [here](https://github.com/beyondessential/tupaia-backlog/issues/1731)